artificial-scrum-master/apps/artificial.scrum.master/Artificial.Scrum.Master.ScrumProjectIntegration/DependencyInjectionScrumProjectIntegrationExtension.cs

```diff
@@ -0,0 +1,38 @@
+using Artificial.Scrum.Master.ScrumProjectIntegration.Features.Projects;
+using Artificial.Scrum.Master.ScrumProjectIntegration.Infrastructure.ApiTokens;
+using Artificial.Scrum.Master.ScrumProjectIntegration.Infrastructure.ScrumServiceHttpClient;
+using Artificial.Scrum.Master.ScrumProjectIntegration.Settings;
+using Artificial.Scrum.Master.ScrumProjectIntegration.Utilities;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Artificial.Scrum.Master.ScrumProjectIntegration
+{
+    public static class DependencyInjectionScrumProjectIntegrationExtension
+    {
+        public static IServiceCollection AddScrumProjectIntegration(
+            this IServiceCollection services,
+            IConfigurationSection scrumManagementServiceSettings)
+        {
+            var settings = new ScrumManagementServiceSettings();
+            scrumManagementServiceSettings.Bind(settings);
+            services.AddSingleton(settings);
```

Comment:
po co to skoro baseUrl przekazujesz w AddHttpClient? Jak dla mnie mógłbyś przekazać całe IConfiguration configuration do metody i w AddHttpClient użyć configuration['...']. Nie potrzebujesz osobnego modelu do tego.